test/test.js

```diff
@@ -98,25 +98,16 @@ describe("live demo", () => {
   }
 
   const loadCmPageAndPrepareSampleContent = async () => {
-    await driver.get('https://codemirror.net/5/');
-    await driver.wait(Until.elementLocated(By.css('.CodeMirror')));
-    const setCodeMirrorContent = 'document.getElementsByClassName("CodeMirror")[0].CodeMirror.setValue("<html><body><h1>live demo</h1><div>This is an tesst</div></body></html>")';
+    await driver.get('https://codemirror.net/');
+    const setCodeMirrorContent = 'view.dispatch(view.state.update({changes: {from: 0, to: view.state.doc.length, insert: "<html><body><h1>live demo</h1><div>This is an tesst</div></body></html>"}}));';
     await driver.executeScript(setCodeMirrorContent);
   }
 
   const loadCkPageAndPrepareContent = async () => {
-    await driver.get('https://ckeditor.com/ckeditor-4/demo/');
-    await driver.wait(Until.elementLocated(By.css('#cke_ckdemo')));
-    const contentIframe = await driver.wait(Until.elementLocated(By.css('iframe.cke_wysiwyg_frame')));
-    await driver.wait(Until.elementLocated(By.css('#cke_1_contents')));
+    await driver.get('https://ckeditor.com/docs/ckeditor5/latest/examples/builds/balloon-editor.html');
+    await driver.wait(Until.elementLocated(By.css('.ck-editor__editable')));
 
-    await driver.switchTo().frame(contentIframe);
-
-    await driver.wait(Until.elementLocated(By.xpath('//h2[text()[contains(.,"Essential")]]')));
-
-    await driver.switchTo().defaultContent();
-
-    const setCkContent = 'while(!(CKEDITOR && CKEDITOR.instances["ckdemo"] && CKEDITOR.instances["ckdemo"].status =="ready")) {}; CKEDITOR.instances["ckdemo"].setData("This is an tesst");';
+    const setCkContent = 'document.querySelector(".ck-editor__editable").ckeditorInstance.data.set("<p>This is an tesst</p>")';
```

Comment:
two comments:
1) we are mixing single and double quotes, would it not be nicer/more consistent to only use one style?
2) this ".ck-editor__editable" string also appears in line 108 above, maybe it should be a constant to avoid the redundancy?

Reply:
1. Added prettier config
2. Made `ck-editor__editable`as variable